ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransaction.java

```diff
@@ -444,7 +444,7 @@ public interface MemorySize {
     int OPTIONAL_SHALLOW_SIZE = 16;
     int KZG_COMMITMENT_OR_PROOF_SIZE = 112;
     int BLOB_SIZE = 131136;
-    int BLOBS_WITH_COMMITMENTS_SIZE = 40;
+    int BLOBS_WITH_COMMITMENTS_SIZE = 48;
```

Comment:
the increment is due to the `int versionId`, that takes 4 bytes, but it is aligned to 8, so in case of future addition there is some free space.
A possible optimization is to make the `versionId` a byte, that will have no effect now, but will leave more free space for future fields.